### PR TITLE
add humble jazzy multi snap monthly check

### PR DESCRIPTION
**Pull request by giusebar**

### CODE REVIEW

File: .github/workflows/snap.yaml [hunk 1]
```diff
@@ -112,7 +121,7 @@ jobs:
                     check_existence rosnode turtlebot3_slam_gmapping 30
                     check_existence rostopic map 30
                     echo "moving the robot"
-                    timeout 5s rostopic pub -r 10 /cmd_vel geometry_msgs/Twist -- '[0.0, 0.0, 0.0]' '[1.0, 0.0, 0.0]'
+                    timeout 5s rostopic pub -r 10 /cmd_vel geometry_msgs/Twist -- '[0.0, 0.0, 0.0]' '[1.0, 0.0, 0.0]' || true
```
Comment (artivis) [inline .github/workflows/snap.yaml:124]: Why this change?

Comment (giusebar) [reply .github/workflows/snap.yaml:124]: because now we execute the bash scripts with [the -e flag](https://github.com/ubuntu-robotics/snap_workflows/blob/main/.github/workflows/snap.yaml#L114), and timeout sends a sigterm after the timeout. So without the || true the script just stops and won't execute the commands after.

**Review by artivis — APPROVED** [review]
